app/api/domains/cho.py

```diff
@@ -397,8 +397,15 @@ async def handle(self, player: Player) -> None:
                         # use player mode if not specified
                         mode_vn = player.status.mode.as_vanilla
 
+                    # parse the mods from regex
+                    if r_match["mods"] is not None:
```

Comment:
same as other one

Reply:
well I did it the same way it's on other parts in cho.py, just copied existing if statements and put them in place